script/main_test.py

```diff
@@ -1,25 +1,57 @@
 import unittest
+import main
 
 
 structure_to_test =[{"phone":"04-56-18-88-34","cell":"06-74-93-14-75"},{"phone":"17533450","cell":"67583818"},{"phone":"075 831 68 55","cell":"075 292 22 27"},{"phone":"(394)-381-6746","cell":"(361)-356-8993"},{"phone":"(00) 1170-0136","cell":"(32) 7367-6536"}]
 
 
-# class TestRemovesSpecialCharacters(unittest.TestCase):
+class TestRemovesSpecialCharacters(unittest.TestCase):
 
-#   def test_remove_special_characters(self):
-#     dict1 = {
-#       "phone":"04-56-18-88-34",
-#       "cell":"06-74-93-14-75"
-#     }
-#     list_dict = [dict1]
+  def test_remove_special_characters(self):
```

Comment:
This function should looks like this:
```python
def test_remove_special_characters(self):
    self.assertEqual(main.clean_phone_number("04-56-18-88-34"), "0456188834")
```

But the problem here is that there is no such function `clean_phone_number`. Instead you have only function `remove_special_characters_from_phone_numbers` which is hard to test.
Refactor the code in the main.py. Create this new function and use it inside `remove_special_characters_from_phone_numbers` to clean phone numbers.